cogboard-app/src/test/kotlin/com/cognifide/cogboard/config/storage/VolumeStorageFactoryTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.cognifide.cogboard.config.storage
+
+import com.cognifide.cogboard.config.ConfigType
+import com.cognifide.cogboard.storage.VolumeStorageFactory
+import io.vertx.core.json.JsonObject
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.io.File
+
+internal class VolumeStorageFactoryTest {
+
+    private val boardPath = VolumeStorageFactoryTest::class.java.getResource("/board").path
+    private val configThatShouldNotExist = "$boardPath/initial-config-for-VolumeStorageFactoryTest-test.json"
```

Comment:
why not simple initalData ?

Reply:
This will be changed completely we don't need any files as we have them already.